packages/patternfly-4/react-core/src/components/Dropdown/Item.js

```diff
@@ -31,8 +29,7 @@ Item.defaultProps = {
   isHovered: false,
   component: 'a',
   isDisabled: false,
-  href: '#',
-  role: 'menuitem'
+  href: '#'
```

Comment:
Also need to update the typescript definitions for Item, KebabToggle, and Separator?

Reply:
KebabToggle extends DropdownToggle and Separator Extends DropdownItem.